### PR TITLE
Added toggle button for theme switch

### PR DESCRIPTION
**Pull request by deytulsi18**
Added toggle button for theme switch, This PR is with reference to #514

### CODE REVIEW

**Comment by deytulsi18** [comment]
Please check and merge this PR @smaranjitghose

**Comment by smaranjitghose** [comment]
toggllleee
Please check the preview before giving the PR.
The toggle's size should be adjusted so that it conforms to that of the other tabs on the nav bar

File: assets/js/app.js [hunk 1]
```diff
@@ -18,6 +18,23 @@ let header = $(`
...
+          <label for="switch-1" class="theme-switch">
+        <input class="toggle-checkbox" type="checkbox" id="switch-1" name="theme" />
+        <div class='toggle-slot'>
+            <!-- <div class='sun-icon-wrapper'>
+                    <div class="iconify sun-icon" data-icon="feather-sun" data-inline="false"> </div>
+                </div> -->
+            <div class='toggle-button-1 toggle-button'>
+                <span class="toggle--feature"></span>
+            </div>
+            <!-- <div class='moon-icon-wrapper'>
```
Comment (anushbhatia) [inline assets/js/app.js:32]: Remove the code from in comments

**Comment by smaranjitghose** [comment]
The toggle should work on click on the entire region. Rather than just the ends of the toggle

File: assets/js/app.js [hunk 2]
```diff
@@ -76,6 +93,89 @@ $(function () {
...
+    let a = localStorage.getItem("theme");
+    document.documentElement.setAttribute("data-theme", a); // setting the initial theme to light
+
+    if (localStorage.getItem("theme") === "dark") {
+      checkbox.checked = true;
+    }
+
+    // selecting the footer text & footer icons
+    const footerText = document.querySelector('.footer-text p');
+    const footerIcons = document.querySelectorAll('.sub-footer-icon');
```
Comment (anushbhatia) [inline assets/js/app.js:112]: What is the use of footer Icon constant if they are not implemented in code anywhere

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Make the theme working for all pages

**Comment by anushbhatia** [comment]
Put a comment like

``` terminal
/* 
---------------------------------
         TEXT ANIMATIONS
----------------------------------
*/
```
so the file is more readable

**Comment by deytulsi18** [comment]
So I want to say something 😐, I think maybe I am not able enough to do this task properly. I would ask you to kindly assign this to someone able.
I spent my whole two days working constantly for creating button and making it proper, I even missed two or three online collge classes to complete it🙁. 

I guess I am not able enough pl pardon me 😔

**Comment by anushbhatia** [comment]
Its okay

**Pull request closed without merging**